src/main/scala/dpla/ingestion3/executors/MappingExecutor.scala

```diff
@@ -68,23 +66,24 @@ trait MappingExecutor extends Serializable {
     val oreAggregationEncoder: ExpressionEncoder[Row] = RowEncoder(model.sparkSchema)
 
     // Load the harvested record dataframe
-    val harvestedRecords: DataFrame = spark.read.avro(dataIn).repartition(1024)
+    val harvestedRecords: DataFrame = spark.read.avro(dataIn)
```

Comment:
IIRC I think this was in there to make Nara not be 1 partition based on the input data (sequencefiles built from their tar files). I think with our current harvester, which creates avros, it's totally unneeded.